### PR TITLE
Universal class for subreddit-specific toggle buttons

### PR DESCRIPTION
**Pull request by githue**
subredditInfo.js relied on filteReddit.js to apply the correct style to the subscribe button. This PR creates a universal class based on fancy-toggle-button which is module-independent, and removes doubled-up CSS.

I named the class res-subreddit-btn because all the relevant buttons perform subreddit actions: subscribe, shortcut, dashboard, filter. This doesn't touch the +friends button because that uses reddit's class names.

### CODE REVIEW

**Comment by andytuba** [comment]
awyeah.jpg

File: lib/core/res.css [hunk 1]
```diff
@@ -790,6 +790,21 @@ p.moduleListing {
...
 }
+/* Universal subreddit toggle button (subscribe, filter, shortcut, etc) */
+/* Borrowed from fancty-toggle-button */
+.res-subreddit-btn {
+	font: bold 10px verdana, sans-serif; /* allow line-height to be inherited */
+	color: white;
+	margin: 0 5px 5px 0;
+	padding: 1px 6px;
+	border: 1px solid rgb(68,68,68);
+	background: rgb(123,184,80) url(https://redditstatic.s3.amazonaws.com/bg-button-add.png) 0 100% repeat-x;
```
Comment (andytuba) [inline lib/core/res.css]: I'd love to take this opportunity to remove that background-url.

**Comment by githue** [comment]
I'll add the old class back as a legacy thing, the themes I checked didn't show any issues.

File: lib/core/res.css (hunk 1 above)
Comment (githue) [inline lib/core/res.css]: We can just replace it with linear-gradient, I don't know why there was an image in the first place.

**Comment by githue** [comment]
Naut is back to normal. I went through a bunch of style-heavy subs and found two that still look different: /r/askreddit and /r/wow. The former can't be helped because it uses inelegant techniques to position the buttons, and the latter just looks bigger than it should because it's forcing the line-height.

Also replaced the image URL with a linear-gradient.

**Comment by githue** [comment]
Is there a better name for the class instead of res-subreddit-button? What about res-fancy-toggle-btn to convey that it's meant to emulate reddit's buttons?

**Comment by erikdesjardins** [comment]
+1 for res-fancy-toggle-button

**Comment by andytuba** [comment]
res-fancy-toggle-button would be cool, a little more general-use than subreddit-button.

**Pull request merged**